content/docs/apps/using-ssh.md

```diff
@@ -20,3 +20,24 @@ in order to debug your application.
 You can interact directly with the services bound to your application via [port-forwarding](https://docs.cloudfoundry.org/devguide/deploy-apps/ssh-services.html). This allows you to access those services using native clients on your local machine. The [Service Connect plugin](https://github.com/18F/cf-service-connect#readme) makes this even easier.
 
 `cf ssh` uses port 2222. If your network blocks port 2222, you may receive an error message such as `Error opening SSH connection`.
+
+
+## SSH-Related Questions
+
+*Q*: When I SSH to my application and I try to run `python`  I get `ImportError: No module named site`. How should I set that up?
```

Comment:
On https://cloud.gov/docs/apps/logs/#error-messages we have a section for error messages. This page could have something similar - it already has one error case (If your network blocks port 2222, you may receive an error message such as Error opening SSH connection.) so the two error cases could be together in a section.